bom/pom.xml

```diff
@@ -20,22 +20,22 @@
   <properties>
 
     <!-- Modules Versions -->
-    <daisy.calabash-adapter.version>2.0.5</daisy.calabash-adapter.version>
+    <daisy.calabash-adapter.version>2.0.6-SNAPSHOT</daisy.calabash-adapter.version>
     <daisy.calabash-custom-steps.version>2.0.0</daisy.calabash-custom-steps.version>
     <daisy.common-utils.version>2.1.3</daisy.common-utils.version>
-    <daisy.framework-core.version>3.1.1</daisy.framework-core.version>
-    <daisy.framework-persistence.version>2.0.4</daisy.framework-persistence.version>
-    <daisy.framework-volatile.version>2.0.2</daisy.framework-volatile.version>
+    <daisy.framework-core.version>3.2.0-SNAPSHOT</daisy.framework-core.version>
+    <daisy.framework-persistence.version>2.0.5-SNAPSHOT</daisy.framework-persistence.version>
+    <daisy.framework-volatile.version>2.0.3-SNAPSHOT</daisy.framework-volatile.version>
     <daisy.launcher.version>1.0.0</daisy.launcher.version>
     <daisy.logging-activator.version>2.0.2</daisy.logging-activator.version>
     <daisy.logging-appender.version>1.1.1</daisy.logging-appender.version>
     <daisy.modules-registry.version>2.0.1</daisy.modules-registry.version>
-    <daisy.persistence-derby.version>2.0.1</daisy.persistence-derby.version>
-    <daisy.persistence-mysql.version>2.0.0</daisy.persistence-mysql.version>
+    <daisy.persistence-derby.version>2.0.2-SNAPSHOT</daisy.persistence-derby.version>
+    <daisy.persistence-mysql.version>2.0.1-SNAPSHOT</daisy.persistence-mysql.version>
     <daisy.push-notifier.version>2.0.1</daisy.push-notifier.version>
-    <daisy.updater-wrapper.version>1.0.1</daisy.updater-wrapper.version>
-    <daisy.webservice.version>2.1.2</daisy.webservice.version>
-    <daisy.webservice-utils.version>2.2.2</daisy.webservice-utils.version>
+    <daisy.updater-wrapper.version>1.0.2-SNAPSHOT</daisy.updater-wrapper.version>
+    <daisy.webservice.version>2.1.3-SNAPSHOT</daisy.webservice.version>
+    <daisy.webservice-utils.version>3.0.0-SNAPSHOT</daisy.webservice-utils.version>
```

Comment:
I do not see API change in the `webservice` module, am I correct that the major change is due to change in the web service payload?

Reply:
I don't remember TBH. Might be a mistake.

Reply:
I remember now. It's because I changed the `Properties` class into an enum.

Reply:
ok!